### DISCUSSION TITLE
Futaba S-FHSS Automatically binding (1.3.3.7)

### DISCUSSION THREAD

**Discussion by Yummyhole** [pascallanger/DIY-Multiprotocol-TX-Module#703] [Q&A] [answered]
I own a Radiomaster Tx16s mark I and II.  I’ve have both on EdgeTX 2.7.1.  I’ve never had to use the Futaba S-FHSS protocol before.  I recently purchased an OMP M1 with built in S-FHSS receiver and a Radiomaster WT-01 and am using S-FHSS to bind to it.  I noticed that when using the  S-FHSS protocol that is automatically binds.  Everything seems to work ok when it does but when I started researching, I am seeing others are not seeing this behavior.  If I put either the M1 or the WT-01 into bind mode and attempt to manually bind it never completes.   I am using version 1.3.3.7 of the multi module firmware.  I have fined tuned the protocol for each radio.  My concern is that is is automatically binding on its own and when I attempt to manually bind it fails. This may be the expected behavior just want to make sure.

**Comment by pascallanger** [comment, marked as answer]
I've added this line in the protocol details: "This protocol does not use bind on the TX side. The RX attaches to the first S-FHHSS TX around it when the bind button is pressed."
Also version 1.3.3.14 exits bind immediately if you press the bind button on the TX side. I don't remember in which version this has been added but it's there in 1.3.3.14

**Reply by Yummyhole** [reply]
Thank you for for the clarification and quick response.  Have a great day.
